src/cmd/run.go

```diff
@@ -71,6 +73,12 @@ func init() {
 		"",
 		"Run command inside a toolbox container for a different operating system release than the host")
 
+	flags.IntVarP(&runFlags.preserve_fds,
```

Comment:
Nitpick: with [flags.IntVar](https://pkg.go.dev/github.com/spf13/pflag#IntVar) (ie., without the trailing `P`) ...

Reply:
Done.

Reply:
Thanks.